DESCRIPTION

```diff
@@ -18,7 +18,9 @@ Imports:
     R6 (>= 2.0.0),
     stringi (>= 0.3.0),
     jsonlite (>= 0.9.16),
-    httpuv (>= 1.2.3)
+    httpuv (>= 1.2.3),
+    feather,
```

Comment:
I'd hope to keep feather off of `imports` and in `suggests`. See how I handle the `htmlwidgets` package in this PR: https://github.com/trestletech/plumber/pull/82/files

We could just suggest and then, if a user goes to use a feature that requires a particular package and doesn't have it, we'd abort at runtime.